indico/modules/events/papers/controllers/base.py

```diff
@@ -85,3 +86,14 @@ def _check_paper_protection(self):
         privileges than the generic "can access".
         """
         return self.contribution.can_access(session.user)
+
+
+class RHSubmitRevisionBase(RHPaperBase):
```

Comment:
I'd keep this in the other file. While a "base" it's so specific to these two RHs that do almost the same; everything else in here is much more generic.